docs/How-We-Work/Team-Composition.md

```diff
@@ -58,3 +57,5 @@
 * Timothy Spencer, 18F, consulting engineer
 * Abbey Kos, 18F, research consultant
 * Randy Hart, 18F, acquisition consultant
+* Spencer Hilvitz, Raft, front end engineer
```

Comment:
@kniz-raft  can you move Spencer and Romelo to the top of this list as most recent?

Reply:
Done! I also added John Willis to current team.